README.md

```diff
@@ -2,6 +2,10 @@
 
 This is the engine for [Webspaces](https://webspaces.space), which allows you to create 3D worlds using just HTML.
 
+## Notice
```

Comment:
sorry to be pedantic, but we can probably drop this header

Reply:
Sure! its alright, i need to learn the job properly if i wanna help you right? XD